app/services/hybrid_transcription_service.py

```
@@ -0,0 +1,361 @@
+                    "service": "runpod",
+                    "audio_file_size_mb": file_size_mb if 'file_size_mb' in locals() else None,
+                    "success": True
+                }
+            else:
+                logger.error(f"RunPod transcription returned no text for {video_id}")
+                return None, {
+                    "method": "runpod_whisper",
+                    "error": "RunPod returned no transcription text",
+                    "audio_file_size_mb": file_size_mb if 'file_size_mb' in locals() else None,
+                    "success": False
+                }
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)